core/arch/arm/mm/mobj.c

```diff
@@ -965,7 +967,9 @@ struct mobj *mobj_seccpy_shm_alloc(size_t size)
 	if (tee_mmu_add_rwmem(utc, &m->mobj, &va) != TEE_SUCCESS)
 		goto bad;
 
-	if (!tee_pager_add_uta_area(utc, va, size))
+	m->fobj = fobj_rw_paged_alloc(ROUNDUP(size, SMALL_PAGE_SIZE) /
+				      SMALL_PAGE_SIZE);
+	if (tee_pager_add_uta_area(utc, va, m->fobj))
 		goto bad;
 
```

Comment:
Add `fobj_put(m->fobj);` here.

Reply:
You're right. Which commit is this?

Reply:
In `core: change tee_pager_add_uta_area() arguments`, [here](https://github.com/OP-TEE/optee_os/pull/2801/commits/81c1ba1c9525b5d056f3adb1f7030e707c4409bb#diff-1b5dd72b044761b82ecce76d98e0bdf9R970).

Reply:
No a `fobj_put()` isn't needed here, the pointer is saved at later put in `mobj_seccpy_shm_free()`